### DISCUSSION TITLE
Example of Wiki

### DISCUSSION THREAD

**Discussion by rootsmusic** [community/community#23875] [New to GitHub]
<p>In a repository, its navigation has a Wiki section.  Where can I find a Wiki example that has content?  Thanks.</p>

**Comment by airtower-luna** [comment]
<p>Are you looking for an example of something specific, or really just about anything? In the latter case, here’s one of mine: <a href="https://github.com/airtower-luna/mod_gnutls/wiki">https://github.com/airtower-luna/mod_gnutls/wiki</a></p>

**Comment by rootsmusic** [comment]
<p>Thanks, <a class="mention" href="/airtower-luna">@airtower-luna</a>.  I assume that the Wiki is intended for multiple contributors, so I’d like to see one that’s active to better understand how it can help a repository.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Well, the uBlock Origin one is moderately busy, and mostly home to documentation:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/gorhill/uBlock/wiki">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/gorhill/uBlock/wiki" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/5814919/181079798-d15ca486-00b3-445f-b311-f9bd4750dfb7.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/5/454d7b031d6b2d718672f17fd2535d4eef0ae5a1_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/5/454d7b031d6b2d718672f17fd2535d4eef0ae5a1_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/5/454d7b031d6b2d718672f17fd2535d4eef0ae5a1.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/5/454d7b031d6b2d718672f17fd2535d4eef0ae5a1_2_10x10.png"></div>

<h3><a href="https://github.com/gorhill/uBlock/wiki" target="_blank" rel="noopener">Home · gorhill/uBlock Wiki</a></h3>

  <p>uBlock Origin - An efficient blocker for Chromium and Firefox. Fast and lean. - Home · gorhill/uBlock Wiki</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>In the end it’s simply a place to share information mostly as Markdown (and possibly images). What is relevant and helpful depends on your project. :slightly_smiling_face:</p>
